### ISSUE TITLE
QA_EXPT-9 & EXPT-10_Introduction and Theory_content-is-same

### ISSUE THREAD

**Issue by BSravanthi** [Virtual-Labs/analog-cmos-vlsi-circuit-design-iiith#28]
Defect Description :
In the Introduction and Theory page of 9th and 10th experiments of this lab, found that the content is same.

Actual Result :
In the Introduction and Theory page of 9th and 10th experiments of this lab, found that the content is same.

Environment :
OS: Windows 7, Ubuntu-16.04,Centos-6
Browsers:Firefox-42.0,Chrome-47.0,chromium-45.0
Bandwidth : 100Mbps
Hardware Configuration:8GBRAM ,
Processor:i5

Attachments:
![qa_oe_avlsi_i25](https://cloud.githubusercontent.com/assets/13479177/26664338/60647600-46ae-11e7-9c0e-cd3f6122ca14.png)

**Comment by sravanthimodepu** [comment]
Yes.. The 9th & 10th experiments belongs to the same experiment. So the content is same, but the simulations are different. Found the same from the vlab.co.in page.

**Comment by BSravanthi** [comment]
Need to contact lab developer. The issue will be opened till we get the response.
